src/Stack/Ghci.hs

```diff
@@ -532,11 +531,10 @@ figureOutMainFile bopts mainIsTargets targets0 packages = do
             T.pack (toFilePath mainIs)
     candidateIndices = take (length candidates) [1 :: Int ..]
     userOption = do
-      putStr "Specify main module to use (press enter to load none): "
-      option <- getLine
+      option <- prompt "Specify main module to use (press enter to load none): "
       let selected = fromMaybe
                       ((+1) $ length candidateIndices)
-                      (readMaybe option :: Maybe Int)
+                      (readMaybe (T.unpack option) :: Maybe Int)
```

Comment:
Not a big deal, but we could use the `Data.Text.Read.decimal` function for this.